R/loadLm.R

```diff
@@ -361,7 +361,7 @@ resampleCoefficients.lm <- function(fit) {
 #' @family simulateSolute
 simulateSolute.loadLm <- function(load.model, flux.or.conc=c("flux","conc"), newdata, 
                                   method=c("parametric", "non-parametric"), from.interval=c("confidence", "prediction"), rho, ...) {
-  
+  s.hat=NULL
```

Comment:
line 364 in loadLm.R; I removed this and the note for this section of the code no longer appears.